app/workers/users_worker.rb

```diff
@@ -0,0 +1,12 @@
+class UsersWorker
+  include Sidekiq::Worker
+
+  def perform bill_id
+    bill = Bill.find bill_id
+    user = bill.user
+    UserMailer.reservation_activation(user, bill).deliver_now
+  rescue StandardError
+    AdminMailer.reservation_error(bill_id).deliver_now
+    raise NameError
```

Comment:
T nghỉ nên bắt error nào thì raise trả error đấy 
```suggestion
  rescue StandardError => e
    AdminMailer.reservation_error(bill_id).deliver_now
    raise e
```